database/influxdb/influxdb_retention_policy.py

```diff
@@ -0,0 +1,234 @@
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: influxdb_retention_policy
+short_description: Manage InfluxDB retention policies
+description:
+    - Manage InfluxDB retention policies
+version_added: 2.1
+author: "Kamil Szczygiel (@kamsz)"
+requirements:
+    - "python >= 2.6"
+    - "influxdb >= 0.9"
+options:
+    hostname:
+        description:
+            - The hostname or IP address on which InfluxDB server is listening
+        required: True
+    username:
+        description:
+            - Username that will be used to authenticate against InfluxDB server
+        default: root
+    password:
+        description:
+            - Password that will be used to authenticate against InfluxDB server
+        default: root
+    port:
+        description:
+            - The port on which InfluxDB server is listening
+        default: 8086
+    database_name:
+        description:
+            - Name of the database where retention policy will be created
+        required: True
+    policy_name:
+        description:
+            - Name of the retention policy
+        required: True
+    duration:
+        description:
+            - Determines how long InfluxDB should keep the data
+        required: True
+    replication:
+        description:
+            - Determines how many independent copies of each point are stored in the cluster
+        required: True
+    default:
+        description:
+            - Sets the retention policy as default retention policy
+        required: True
+'''
+
+EXAMPLES = '''
+# Example influxdb_database command from Ansible Playbooks
+- name: create 1 hour retention policy
+    influxdb_retention_policy:
+      hostname: "{{influxdb_ip_address}}"
+      database_name: "{{influxdb_database_name}}"
+      policy_name: test
+      duration: 1h
+      replication: 1
+
+- name: create 1 day retention policy
+    influxdb_retention_policy:
+      hostname: "{{influxdb_ip_address}}"
+      database_name: "{{influxdb_database_name}}"
+      policy_name: test
+      duration: 1d
+      replication: 1
+
+- name: create 1 week retention policy
+    influxdb_retention_policy:
+      hostname: "{{influxdb_ip_address}}"
+      database_name: "{{influxdb_database_name}}"
+      policy_name: test
+      duration: 1w
+      replication: 1
+
+- name: create infinite retention policy
+    influxdb_retention_policy:
+      hostname: "{{influxdb_ip_address}}"
+      database_name: "{{influxdb_database_name}}"
+      policy_name: test
+      duration: INF
+      replication: 1
+'''
+
+RETURN = '''
+#
```

Comment:
Could you please add this in the RETURN?

```#only defaults```

We need to work on doing this better so these defaults get inserted when the docs are generated. That placeholder will help once that happens.